pkg/beholder/config.go

```diff
@@ -18,20 +18,40 @@ type Config struct {
 	EmitterExportTimeout time.Duration
 	// Batch processing is enabled by default
 	// Disable it only for testing
-	EmitterBatchProcessor bool
+	EmitterBatchProcessor      bool
+	EmitterExporterRetryConfig RetryConfig
 	// OTel Trace
-	TraceSampleRatio  float64
-	TraceBatchTimeout time.Duration
-	TraceSpanExporter sdktrace.SpanExporter // optional additional exporter
+	TraceSampleRatio         float64
+	TraceBatchTimeout        time.Duration
+	TraceSpanExporter        sdktrace.SpanExporter // optional additional exporter
+	TraceExporterRetryConfig RetryConfig
 	// OTel Metric
-	MetricReaderInterval time.Duration
+	MetricReaderInterval      time.Duration
+	MetricExporterRetryConfig RetryConfig
 	// OTel Log
 	LogExportTimeout time.Duration
 	// Batch processing is enabled by default
 	// Disable it only for testing
 	LogBatchProcessor bool
 }
 
+type RetryConfig struct {
+	// Enabled indicates whether to not retry sending batches in case of
+	// export failure.
+	Enabled bool
```

Comment:
It looks like we could omit this explicit enabled field, if we recognized an empty/zero-value struct as disabled. If that is too implicit, another option would be `*RetryConfig` pointer fields :shrug:

Reply:
Addressed here https://github.com/smartcontractkit/chainlink-common/pull/867/commits/4cb30150f84112a6d4d228ebc812ae0600d000ba